encoding/preparation.md

```diff
@@ -275,8 +275,18 @@ adding it to the system PATH is recommeded.
 When working with WEB and TV sources,
 you will most likely have only lossy audio available.
 The most common codecs here are AC-3, E-AC-3 and AAC.
-Lossily compressed files should generally not be reencoded—the proper way to handle them
+**Lossily compressed audio
+should generally not be reencoded**—the proper way to handle them
 is to remux (i.e. copy) them to the final file.
+Reencoding lossy audio into a different lossy audio codec
+degrades the audio quality
```

Comment:
It doesn't, really. If you take a high quality lossy file and reencode it again in high quality, there won't be any audible difference.
https://bernholdtech.blogspot.com/2013/03/Nine-different-audio-encoders-100-pass-recompression-test.html
As somebody on #encoding-autism rightfully mentioned, this test mostly tests how immune a codec is to its own artifacts, but it still shows that even mulitple reencodings won't hurt.

Reply:
I specifically mentioned using a different audio codec, so that test is barely relevant for my argument. Reencoding using the same codec with lower bitrate isn't mentioned anywhere.

However, the point needs to be made that reencoding 140kbps eac3 into 140kbps aac doesn't help the slightest besides maybe hardware compatibility (I don't actually know the hardware support of eac3).

Reply:
I'm not saying reencoding with the same codec should be mentioned. I'm just saying that reencoding lossy audio isn't as terribly harmful as many picture it.

Reply:
Have you checked out my updated wording? It's a bit hard to spot due to the rebase, but it's [here](https://github.com/Irrational-Encoding-Wizardry/guide.encode.moe/compare/e672aa4c08ffeb9bf99567fbf5590953fcabe7f2..96bdf224254192a61f885b50537409e70dfa4c47#diff-d32a06396d1dd04f5fa407edca2e9180R311).

If think that's okay, I'd like to merge this PR for now, but you may of course also work on the lossless or lossy section. I don't have any particular goals for that myself, though.